workflow/controller/controller.go

```diff
@@ -223,8 +223,8 @@ func (wfc *WorkflowController) Run(ctx context.Context, wfWorkers, workflowTTLWo
 	defer wfc.podQueue.ShutDown()
 	defer wfc.podCleanupQueue.ShutDown()
 
-	log.WithField("version", argo.GetVersion().Version).Info("Starting Workflow Controller")
-	log.Infof("Workers: workflow: %d, pod: %d, pod cleanup: %d", wfWorkers, podWorkers, podCleanupWorkers)
+	log.WithFields(log.Fields{"version": argo.GetVersion().Version, "defaultRequeueTime": GetRequeueTime()}).Info("Starting Workflow Controller")
+	log.WithFields(log.Fields{"workflow": wfWorkers, "workflowTtl": workflowTTLWorkers, "pod": podWorkers, "podCleanup": podCleanupWorkers}).Infof("Current Worker Numbers")
```

Comment:
I like to use train-wreck pattern for this now as I think it is easier to read:

```golnga
log.WithField("wfWorkers", wfWorkers).
  WithField("workflowTTLWorkers", workflowTTLWorkers).
  Info("this is the config")
```

Reply:
Done